lib/serialization/BinaryMiddleware.js

```diff
@@ -0,0 +1,341 @@
+/*
+	MIT License http://www.opensource.org/licenses/mit-license.php
+*/
+
+"use strict";
+
+const SerializerMiddleware = require("./SerializerMiddleware");
+
+/*
+Format:
+
+File -> Section*
+
+Section -> NullsSection |
+					 F64NumbersSection |
```

Comment:
indent?

Reply:
github displays this in a weird way. The editor should disable tab as 2 spaces, as noted in the .editorconfig

Reply:
Ahh yep, github did manage to display tabs correctly until this day... 🙄